source/SkiaScene/SkiaScene.TouchManipulation/SceneGestureRenderingResponder.cs

```diff
@@ -28,13 +28,14 @@ public int MaxFramesPerSecond
             {
                 _maxFramesPerSecond = value;
                 _minGestureDuration = TimeSpan.FromMilliseconds(1000d / _maxFramesPerSecond);
-
             }
         }
 
         protected override void TouchGestureRecognizerOnPan(object sender, PanEventArgs args)
         {
-            base.TouchGestureRecognizerOnPan(sender, args);
+            if (args.TouchActionType == TouchActionType.Moved)
```

Comment:
This looks like a reasonable check. Maybe I would just move both of those checks to the base class `SceneGestureResponder` because `SceneGestureRenderingResponder` is just extending `SceneGestureResponder` with invalidating logic but those checks are also relevant in the parent class which in theory can be used independently.

Reply:
Sure, I can make that change.

Reply:
Done